### PR TITLE
Let's test ReactiveCocoa for tvOS 📺

### PR DESCRIPTION
**Pull request by dmcrodrigues**
Suggested in #2864. Depends on #2867 to be fully integrated with Travis.

### CODE REVIEW

File: ReactiveCocoa.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2796,6 +3006,191 @@
...
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				INFOPLIST_FILE = ReactiveCocoaTests/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MTL_ENABLE_DEBUG_INFO = YES;
+				PRODUCT_NAME = "$(PROJECT_NAME)Tests";
+				SDKROOT = appletvos;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				TVOS_DEPLOYMENT_TARGET = 9.2;
```
Comment (NachoSoto) [inline ReactiveCocoa.xcodeproj/project.pbxproj]: Can you remove this to inherit the value in the `xcconfig`?

File: ReactiveCocoa.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2796,6 +3006,191 @@
...
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				INFOPLIST_FILE = ReactiveCocoaTests/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MTL_ENABLE_DEBUG_INFO = YES;
+				PRODUCT_NAME = "$(PROJECT_NAME)Tests";
+				SDKROOT = appletvos;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
```
Comment (NachoSoto) [inline ReactiveCocoa.xcodeproj/project.pbxproj]: Ditto

File: ReactiveCocoa.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2796,6 +3006,191 @@
...
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				INFOPLIST_FILE = ReactiveCocoaTests/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MTL_ENABLE_DEBUG_INFO = NO;
+				PRODUCT_NAME = "$(PROJECT_NAME)Tests";
+				SDKROOT = appletvos;
+				TVOS_DEPLOYMENT_TARGET = 9.2;
```
Comment (NachoSoto) [inline ReactiveCocoa.xcodeproj/project.pbxproj]: Ditto

**Comment by NachoSoto** [comment]
One day setting this up won't require a whole copy of a target and scheme. One day...

**Comment by dmcrodrigues** [comment]
I eagerly expect for that day! Thanks for catching this @NachoSoto ✨

**Comment by ikesyo** [comment]
Thanks @dmcrodrigues! I've merged this into `master` with some modifications ([diff](0bcf912...d9dc5a7)). 🚀

**Comment by mdiep** [comment]
Wanna do the same thing for watchOS? 😄

**Comment by ikesyo** [comment]
We cannot have a test target for watchOS unfortunattely. 😢

**Comment by mdiep** [comment]
> We cannot have a test target for watchOS unfortunately.

Why is that?

**Comment by ikesyo** [comment]
Xcode does not have a template for watchOS test target, and also Quick doesn't have watchOS support. See https://github.com/Quick/Quick/issues/333.

**Pull request merged**